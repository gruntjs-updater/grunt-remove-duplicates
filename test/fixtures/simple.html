<!DOCTYPE html>
<html>
<head>
	<title>Test one</title>
	<script type="text/javascript" src="1.js"></script>
	<script src="1.js"></script>
	<script src="1.js"></script>
	<script type="text/javascript" src="1.js"></script>
	<script     type="text/javascript"       src="1.js"    ></script>
	<script type="text/javascript"
		src="1.js">   
	
	</script>
	<script></script>
	<script src=""></script>
	<script src></script>
	<script src="2.js"></script>
	<link rel="stylesheet" href="1.css"/>
	<link href="1.css" rel="stylesheet" type="text/css">
	<link href="2.css" type="text/css">
	<link href="3.css" type="text/css" type="text/css">
	<link rel="stylesheet" 
		href="2.css" type="text/css">
</head>
<body>
	<script type="text/javascript" src="1.js"></script>
	<span>Hello</span>
	<script type="text/javascript" src="2.js"></script>
	<span>World</span>
</body>
</html>