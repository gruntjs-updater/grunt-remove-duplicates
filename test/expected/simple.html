<!DOCTYPE html>
<html>
<head>
	<title>Test one</title>
	<script type="text/javascript" src="1.js"></script>
	<script></script>
	<script src=""></script>
	<script src></script>
	<script src="2.js"></script>
	<link rel="stylesheet" href="1.css"/>
	<link href="2.css" type="text/css">
	<link href="3.css" type="text/css" type="text/css">
</head>
<body>
	<span>Hello</span>
	<span>World</span>
</body>
</html>